tox.ini

```diff
@@ -262,19 +268,24 @@ commands =
 [testenv:test-release]
 basepython = {[testenv:default-python]basepython}
 skip_install = true
+setenv =
+    TWINE_USERNAME = __token__
```

Comment:
No explicit TWINE_REPOSITORY_URL needed?

Reply:
If `TWINE_REPOSITORY_URL` is not set, it defaults to standard PyPI target.